src/excmds.ts

```diff
@@ -2171,6 +2171,128 @@ export function unset(...keys: string[]) {
     config.unset(...target)
 }
 
+/** Include and toggle css style to a web page.
+ *
+ * Syntax: style {mode} [...args]
+ *
+ * @param mode
+ *         ALIASES          MODES
+ *      styleinclude   : style -i
+ *      styledelete    : style -x
+ *      styledisable   : style -d
+ *      styleenable    : style -e
+ *      styletoggle    : style -t
+ *
+ *  - `-i` includes new rule
+ *  - `-x` deletes existing rule
+ *  - `-d` disables rule
+ *  - `-e` enable rule (default: ENABLED)
+ *  - `-t` toggle a rule on/off (default: ON)
+ *
+ * @param args `--name {name} --append --filter {prefix|domain|URL} --css {[complete css block of code]}`
+ *  - `--name` is name of the rule to be saving in config
+ *  - `--append` appends new style to exiting rule. If `--name` is provided on mode `-i` without `--append`option the existing style and filter will be overwritten.
+ * 	- `--filter` argument list is matched with `OR` and should be separated by comma: `,`. eg. `--filter domain.com,https://*,http://website.com/index.html`. If no filter is given, style will be applyed to every page.
+ * 	    . prefix: URLs ending with '*'. eg. `http://*` or `https://www.domain.com/pages/*`
+ * 	    . domain: URL part not containing characters `:` or `/`. eg: www.domain.com
+ * 	    . URL: Full url filter. eg. https://www.domain.com/index.html
+ *  - `--css` is the CSS full code block to be inserted at page when URL matches the filter rules
+ *
+ *  Here are some examples of usage:
+ *
+ * - Include a new css style matching all https pages:
+ *
+ *  `styleinclude --name stylemod --filter https://* --css body {border: 3px solid green;}`
+ *
+ * - Turn off the style included:
+ *
+ *  `styletoggle --name stylemod`
+ *
```

Comment:
Could you document how to view your current style settings? i.e, `viewconfig styletoggles` :)

Reply:
Yes. For sure.